mongorestore/testdata/testdirs/db1/aVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryLongCollectionNameC%24Ax2lHbMK_bh19s4f_JCieskppac.metadata.json

```diff
@@ -0,0 +1 @@
+{"options":{},"indexes":[{"v":{"$numberInt":"2"},"key":{"_id":{"$numberInt":"1"}},"name":"_id_"}],"uuid":"5fa8904aa1b543b0bcc9f789192cc92c","collectionname":"aVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryLongCollectionNameConsistingOfExactlyTwoHundredAndFiftyNineCharacters"}
```

Comment:
Per Slack discussion, the server namespace (\<db\>.\<collection\>) limit is 255 bytes, so the collection name here should be changed from 259 bytes to something like 240-245 bytes -- still hitting the limit to qualify for truncation (238 bytes), but not causing an error on 4.4 with the additional `db1` prefix in the tests.